test/person.json

```diff
@@ -1,155 +1,164 @@
+                "responses": {
+                    "200": {
+                        "description": "person response",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/Person"
+                            }
+                        },
+                        "headers": {
+                            "x-expires": {
+                                "type": "string"
+                            }
+                        }
+                    },
+                    "default": {
+                        "description": "unexpected error",
+                        "schema": {
+                            "$ref": "#/definitions/Error"
+                        }
+                    }
+                }
             }
-          }
         }
-      }
     },
-    "Human": {
-      "properties": {
-        "firstName": {
-          "type": "string"
+    "definitions": {
+        "House": {
+            "required": [
+                "lng",
+                "lat"
+            ],
+            "properties": {
+                "description": {
+                    "type": "string"
+                },
+                "lng": {
+                    "type": "double"
+                },
+                "lat": {
+                    "type": "double"
+                }
+            }
         },
-        "lastName": {
-          "type": "string"
+        "Car": {
+            "required": [
+                "provider",
+                "model"
+            ],
+            "properties": {
+                "provider": {
+                    "type": "string"
+                },
+                "model": {
+                    "type": "string"
+                }
+            }
         },
-        "father": {
-          "$ref": "#/definitions/Human"
+        "Person": {
+            "required": [
+                "login"
+            ],
+            "properties": {
+                "_id": {
+                    "type": "string"
+                },
+                "login": {
+                    "type": "string"
+                },
+                "firstName": {
+                    "type": "string"
+                },
+                "lastName": {
+                    "type": "string"
+                },
+                "houses": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "x-swagger-mongoose": {
+                            "$ref": "#/definitions/House"
+                        }
+                    }
+                },
+                "cars": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "x-swagger-mongoose": {
+                            "$ref": "#/definitions/Car",
+                            "exclude-schema": true
```

Comment:
The idea with `exclude-schema` was mainly just at the document definition level so that you could define a Swagger definition (for use in Swagger UI) but did not want a mongoose schema/model generated for that definition.